script/gen_requirements_all.py

```diff
@@ -7,6 +7,8 @@
 import re
 import sys
 
+from script.manifest.requirements import gather_requirements_from_manifests
```

Comment:
```suggestion
from .manifest.requirements import gather_requirements_from_manifests
```

Reply:
Actually this doesn't work. I think it's because we run `script/gen_requirements_all.py` from root directory.
```
Traceback (most recent call last):
  File "script/gen_requirements_all.py", line 10, in <module>
    from .manifest.requirements import gather_requirements_from_manifests
ImportError: attempted relative import with no known parent package
Exited with code 1
```

Reply:
it because script is not a module.

Reply:
Should we make it one?

Reply:
I don't think we need